depthai_ros_driver/src/pipelines/stereo_with_mobilened_ssd_pipeline.cpp

```diff
@@ -0,0 +1,133 @@
+#include <ros/console.h>
+
+#include <pluginlib/class_list_macros.h>
+#include <depthai_ros_driver/pipeline.hpp>
+#include <depthai_ros_driver/pipeline_common.hpp>
+
+#include <depthai/pipeline/node/ColorCamera.hpp>
+#include <depthai/pipeline/node/DetectionNetwork.hpp>
+#include <depthai/pipeline/node/MonoCamera.hpp>
+#include <depthai/pipeline/node/StereoDepth.hpp>
+#include <depthai/pipeline/node/XLinkOut.hpp>
+
+namespace depthai_ros_driver {
+class StereoWithMobilenetSSDPipeline : public rr::Pipeline {
+    using OpenvVINOVersion = dai::OpenVINO::Version;
+
+public:
+protected:
+    /**
+     * @brief Configuring stereo with detection pipeline
+     */
+    void onConfigure(ros::NodeHandle& nh) {
+        std::string blob_file = "./mobilenet-ssd.blob";
+        int openvino_version = static_cast<int>(_pipeline.getOpenVINOVersion());
+        bool with_depth = true;
+        bool stereo_lrcheck = true;
+        bool stereo_extended = false;
+        bool stereo_subpixel = true;
+        int stereo_confidence_threshold = 200;
+        bool preview_keep_aspect_ratio = false;
+
+        rr::sync_ros_param<bool>(nh, "with_depth", with_depth);
+        rr::sync_ros_param<bool>(nh, "stereo_lrcheck", stereo_lrcheck);
+        rr::sync_ros_param<bool>(nh, "stereo_extended", stereo_extended);
+        rr::sync_ros_param<bool>(nh, "stereo_subpixel", stereo_subpixel);
+        rr::sync_ros_param<int>(nh, "stereo_confidence_threshold", stereo_confidence_threshold);
+        rr::sync_ros_param<std::string>(nh, "blob_file", blob_file);
+        rr::sync_ros_param<int>(nh, "mobilenetssd_openvino_version", openvino_version);
+        rr::sync_ros_param<bool>(nh, "preview_keep_aspect_ratio", preview_keep_aspect_ratio);
```

Comment:
please add "stereo_confidence_threshold" and "preview_keep_aspect_ratio" to depthai_node.launch